azkaban-common/src/test/resources/sample_flow.flow

```diff
@@ -1,3 +1,6 @@
+---
+name: sample_flow
+type: flow
```

Comment:
Is the type here necessary? Doesn't a .flow file always represent a flow type?

Reply:
Originally I thought it was easy to implement because both parent flow and embedded flows could be loaded to azkabanFlow in the same way if we define the same structure. But I think from user's perspective, it would be redundant info if we put type flow here. Same as the flow name. I'll remove it in a separate PR, thanks!

Reply:
@HappyRay Submitted the changes in another PR https://github.com/azkaban/azkaban/pull/1531. Can you take a look?